data/zooroids/zooroids.txt

```diff
@@ -0,0 +1,292 @@
+	"final explode" "blood"
+	description `Oddly shaped like a boomerang, these space creatures are in a constant rotary motion caused by a small internal organ. The motion provides a natural defense against small debris and projectiles that threaten to intercept the creature, which is harmless otherwise.`
+
+outfit "Boomerang Organ"
+	category "Turrets"
+	cost 0
+	thumbnail "outfit/unknown"
+	"mass" 10
+	"outfit space" 10
+	"radar jamming" 30
+	"optical jamming" 20
+	"energy capacity" 100
+	"energy generation" 1
+	"heat generation" 0.5
+
+
+ship "Zooroid" "Zooroid (C)"
+	noun "creature"
+	sprite "ship/zooroids/zooroid-c"
+		"frame rate" 24
+		"random start frame"
+	swizzle 0
+	"never disabled"
+	attributes
+		category "Fighter"
+		"cost" 0
+		"shields" 0
+		"hull" 860
+		"required crew" 1
+		"bunks" 1
+		"mass" 200
+		"drag" 9
+		"heat dissipation" 1.8
+		"fuel capacity" 0
+		"cargo space" 0
+		"outfit space" 0
+		"weapon capacity" 0
+		"engine capacity" 0
+
+		"energy capacity" 1000
+		"energy generation" 100
+		"inscrutable" 1
+
+		"thrust" 6
+		"turn" 80
+
+		"energy protection" 1
+		"heat protection" 1
+		weapon
+			"blast radius" 1600
+			"shield damage" 3000
+			"hull damage" 8000
+			"hit force" 4000
+			"piercing" 0.1
+	explode "flamethrower die" 1
+	"final explode" "fission explosion" 
+	description `This species of space life is found around hotter stars and feeds on solar energy, storing it in four organs that give off the appearance of eyes. Due to all of the internal energy and pressure, these creatures explode violently upon death, which may surprise aggressors.`
+
+effect "fission explosion"
+	sprite "effect/explosion/nuke"
+		"no repeat"
+		"frame rate" 30
+	sound "heavy rocket hit"
+	"lifetime" 15
+	"random angle" 360
+	"random spin" 5
+	"random velocity" 1
+	"velocity scale" .4
+
+
+ship "Astral Cetacean"
+	noun "creature"
+	sprite "ship/astral cetacean/astral"
+		"frame rate" 24
+		"random start frame"
+	swizzle 0
+	"never disabled"
+	attributes
+		category "Transport"
+		"cost" 0
+		"shields" 0
+		"hull" 11000
+		"required crew" 1
+		"bunks" 1
+		"mass" 1200
+		"drag" 15.7
+		"heat dissipation" 0.68
+		"fuel capacity" 200
+		"cargo space" 0
+		"outfit space" 0
+		"weapon capacity" 0
+		"engine capacity" 0
+
+		"hull repair rate" 3
+		"ramscoop" 5 
+		"energy capacity" 2000
+		"energy generation" 160
+
+		"thrust" 24
+		"flare sound" "astral cetacean"
+		"turn" 240
+		"steering flare sound" "astral cetacean"
+
+		"jump fuel" 80
+		"jump speed" .1
+		"hyperdrive" 1
+		"hyperdrive sound" "organ drive"
+		"hyperdrive out sound" "organ drive out"
+		"hyperdrive in sound" "organ drive in"
+
+		"energy protection" 2
+		"heat protection" 2
+		"ion protection" 2
+		weapon
+			"blast radius" 300
+			"shield damage" 3000
+			"hull damage" 1500
+			"hit force" 4500
+	outfits
+		"Astral Mouth"
+	engine 0 0
+	"steering engine" 0 0
+	gun 0 -210
+	leak "blood" 60 60
+	explode "blood" 160
+	"final explode" "blood" 
+	description `Space whale.`
```

Comment:
Didn't want to make this too long as it's not going to be seen in normal gameplay, and I wanted to maintain some mystery about it. Let me know what you think.
```suggestion
	description `This creature stands out from the rest of its kind through its natural ability to jump between systems. The mechanism by which it does this is unknown, and you suspect that attempting to study it would reveal more questions than answers.`
`   Its coloring makes it seem as though it was part of space itself.`
```

Reply:
Hmm. I should say I've always imagined this one to be not specifically of the Vyrmeid species, and is an entity of its own, but lives in the same region. I'd also play down the "descriptive" approach of this and more of a "wonder" explanation of its existence. As in, instead of the hyperspace ability being the main point, its ghastly rare appearance being the main point.

Reply:
How about:
Despite its large size, only the most observant of captains could notice the existence of this creature. Its natural attenuation to the depths of space allow it to travel beyond where most ships are able, and its coloring makes it seem as though it is part of space itself.

Reply:
"Only few captains have had the chance to observe this illustrious creature. Its natural attenuation to the depths of space allows it to travel beyond where most ships are able, and its appearance reflects the both beautifulness and eeriness of space itself."

Reply:
Sounds good!